test/architecture-tests/src/test/java/software/amazon/awssdk/archtests/CodingConventionWithSuppressionTest.java

```diff
@@ -50,7 +50,9 @@ public class CodingConventionWithSuppressionTest {
         Arrays.asList(ArchUtils.classNameToPattern(EmfMetricLoggingPublisher.class),
                       ArchUtils.classNameToPattern(MetricEmfConverter.class),
                       ArchUtils.classNameToPattern(MakeHttpRequestStage.class),
-                      ArchUtils.classNameToPattern("software.amazon.awssdk.services.s3.internal.crt.S3CrtResponseHandlerAdapter")));
+                      ArchUtils.classNameToPattern("software.amazon.awssdk.services.s3.internal.crt.S3CrtResponseHandlerAdapter"),
+                      ArchUtils.classNameToPattern(
```

Comment:
This file follows the logging pattern of the other response transformers, see for example: [FileAsyncResponseTransformer](https://github.com/aws/aws-sdk-java-v2/blob/master/core/sdk-core/src/main/java/software/amazon/awssdk/core/internal/async/FileAsyncResponseTransformer.java#L167)